test_util/compareECL.cpp

```diff
@@ -60,6 +61,29 @@ static void printHelp() {
 }
 
 
+int count_files_case(const std::string rootName){
+
+    std::vector<std::string> extList = {"EGRID", "INIT", "UNRST", "SMSPEC", "RFT"};    
+
+    int count =0;
+
+    for (auto ext : extList){
+
+        std::fstream fileH;
+        std::string fileName = rootName + "." + ext;
+        fileH.open(fileName, std::ios::in);
+
+        if (fileH) {
+            count++;
+        }
+        
+        fileH.close();
+    }
+    
+    return count;
+}
```

Comment:
The way we use the function we only really care if the number of result files is zero or non-zero. Therefore the function could be implemented a bit more briefly as
```C++
bool has_result_files(const std::string& rootName)
{
    std::vector<std::string> extList = { "EGRID", "INIT", "UNRST", "SMSPEC", "RFT" };

    for (const auto& ext : extList) {
        std::ifstream is(rootName + '.' + ext);
        if (is) {
            return true;
        }
    }

    return false;
}
```